net/keepalived/files/usr/libexec/keepalived/hotplug.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/sh
```

Comment:
This is not an executable its a lib so it should go to for example to `/lib/functions` or to `/usr/share/keepalived/`

Reply:
Moved to `/lib/functions/keepalived/hotplug.sh`